src/ruby/pb/test/xds_client.rb

```diff
@@ -113,13 +137,10 @@ def configure(req, _call)
       metadata_value = m.value
       metadata_to_send[rpc][metadata_key] = metadata_value
     end
-    GRPC.logger.info("Configuring new rpcs_to_send and metadata_to_send...")
-    GRPC.logger.info(rpcs_to_send)
-    GRPC.logger.info(metadata_to_send)
     new_rpc_config = RpcConfig.new
-    new_rpc_config.init(rpcs_to_send, metadata_to_send)
+    new_rpc_config.init(rpcs_to_send, metadata_to_send, req['timeout_sec'])
```

Comment:
nit: just pass `req['types']` and remove `rpcs_to_send` local ?

Reply:
Comment addressed